packages/cli/test/mocks/project.ts

```diff
@@ -390,7 +390,7 @@ function exposeSystemEnvs(
 ) {
   const envs: Env = {};
 
-  if (autoExposeSystemEnvs) {
+  if (autoExposeSystemEnvs && target !== 'development') {
     envs['VERCEL'] = '1';
     envs['VERCEL_ENV'] = target || 'development';
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (soft-blocking):** If `target` is not development, do we still want to default `VERCEL_ENV` to "development" here?

Reply:
Since this is a mock, to match the api behavior it should not provide a default `VERCEL_ENV` for dev. I made that decision in https://github.com/vercel/api/pull/27777 because it feels more consistent to just not send any system env vars in dev. But, I'm open to changing the api behavior to make `VERCEL_ENV` and maybe `VERCEL` exceptions.